pangeo-notebook/packages.txt

```diff
@@ -374,9 +374,9 @@ nomkl-1.0
 notebook-6.4.8
 nspr-4.32
 nss-3.74
-numba-0.53.1
+numba-0.55.0
 numcodecs-0.9.1
-numpy-1.22.1
+numpy-1.20.0
```

Comment:
i don't think this drop is a huge concern... but flagging it

Reply:
Is stackstac causing this directly through its requirements? Or is is something more subtle? Going back a few versions of numpy doesn't seems like the right direction.

Reply:
looks like it's actually xarray-spatial: `conda search xarray-spatial --info` 
```
xarray-spatial 0.3.1 pyhd8ed1ab_0
---------------------------------
file name   : xarray-spatial-0.3.1-pyhd8ed1ab_0.tar.bz2
name        : xarray-spatial
version     : 0.3.1
build       : pyhd8ed1ab_0
build number: 0
size        : 9.6 MB
license     : MIT
subdir      : noarch
url         : https://conda.anaconda.org/conda-forge/noarch/xarray-spatial-0.3.1-pyhd8ed1ab_0.tar.bz2
md5         : 91ac01eb062acda195256a37cdec585b
timestamp   : 2022-01-10 14:10:54 UTC
dependencies: 
  - dask-core
  - datashader
  - numba
  - numpy >=1.7,<=1.20
```
if you want to open an issue over there, we could comment out xarray-spatial and re-lock....

Reply:
Looks like they just have to make a new release. Let's see if that happens quick and we can just re-lock.